string num_req
---
string num_res
